common/src/theme/BlogPostItem/Header/styles.module.css

```
@@ -0,0 +1,13 @@
+.postHeader {
```

Stylelint (17.14.0)

[error] Expected class selector ".postHeader" to be kebab-case (selector-class-pattern)

(selector-class-pattern)